### PR TITLE
Added a of Gen AI landing page

### PR DESCRIPTION
**Pull request by AlexKoff88**

### CODE REVIEW

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+In this case, you can load the converted model in OpenVINO representation directly from the disk:
+
+.. code-block:: python
+
+    model_id = "llama_openvino"
+    model = OVModelForCausalLM.from_pretrained(model_id)
+
+
+To select inference device use `.to()` method. The device naming convention is the same as in OpenVINO native API:
+
```
Comment (helena-intel) [inline docs/articles_en/openvino_workflow/gen_ai.md:99]:
```suggestion
By default, inference will run on CPU. To select a different inference device, for example GPU, add `device="GPU"` to the `.from_pretrained()` call. To switch to a different device after the model has been loaded, use the `.to()` method. The device naming convention is the same as in OpenVINO native API:

```

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+OpenVINO toolkit could be used to accelerate Generative AI use cases as those mostly rely on model inference. When it comes to Generative Models, OpenVINO supports:
+
+* Conversion, optimization and inference for Text, Image and Audio generative models (for example, llama2, MPT, OPT, Stable Diffusion, Stable Diffusion XL, etc.) 
+* Int8 weights compression for text generation models 
+* Reduced storage format (fp16 precision for non-compressed models and int8 for compressed models) 
+* Inference on CPU and GPU platforms, including Intel integrated and discrete ARC GPUs 
+ 
+
+There are two main paths how OpenVINO could be used to run Generative AI use cases:
```
Comment (helena-intel) [inline docs/articles_en/openvino_workflow/gen_ai.md:13]: I don't know if we want to complicate this document, and I don't know the status of that for LLMs, but for stable diffusion, toch.compile() is also a good option.

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+     - OpenVINO
+   * - Inference code
+     - Hugging Face based
+     - Custom inference pipelines
+   * - Time to inference
+     - Slow
+     - Fast
+   * - Additional dependencies
+     - As much as Hugging Face has
+     - Numpy
```
Comment (helena-intel) [inline docs/articles_en/openvino_workflow/gen_ai.md:51]:
```suggestion
   * - Additional Python dependencies
```
openvino-telemetry is also a dependency of OpenVINO runtime Python API. And for C++, we require some libraries to be installed too. These libraries are included in the Python wheels, but when not using wheels they need to be installed.

File: docs/articles_en/openvino_workflow/gen_ai.md (hunk 2 above)
Comment (AlexKoff88) [reply docs/articles_en/openvino_workflow/gen_ai.md:13]: It is experimental for LLMs still from what I know. But we can improve this in the next version when it is more mature.

File: docs/articles_en/openvino_workflow/gen_ai.md (hunk 3 above)
Comment (AlexKoff88) [reply docs/articles_en/openvino_workflow/gen_ai.md:51]: In fact, telemetry is a part of OpenVINO distribution that is just extracted into a separate package for convenience. I agree that it does not sound fully correct. Maybe we should rephrase it to "lightweight (e.g. numpy, boost,, etc.)". @yury-gorbachev, what do you think?

**Comment by AlexKoff88** [comment]
@tsavina, please take a look

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+* Create a Python environment
+* Install Optimium-Intel running:
+
+.. code-block:: console
+
+    pip install optimum[openvino,nncf]
+
+
+To start using OpenVINO as a backend for Hugging Face you should change original Hugging Face code in two places.
```
Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:74]:
```suggestion
To start using OpenVINO as a backend for Hugging Face, change the original Hugging Face code in two places:
```

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+    -from transformers import AutoModelForCausalLM
+    +from optimum.intel. import OVModelForCausalLM
+
+    model_id = "meta-llama/Llama-2-7b-chat-hf"
+    -model = AutoModelForCausalLM.from_pretrained(model_id)
+    +model = OVModelForCausalLM.from_pretrained(model_id, export=True)
+
+
+After that, you can call ``.save_pretrained()`` method to save model to the folder in the OpenVINO Intermediate Representation and use it further.
```
Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:86]:
```suggestion
After that, you can call the ``.save_pretrained()`` method to save the model to the folder in the OpenVINO Intermediate Representation and use it further.
```

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+
+.. code-block:: python
+
+    model.save_pretrained(model_dir)
+
+
+Alternatively, you can download and convert model using CLI interface: ``optimum-cli export openvino --model meta-llama/Llama-2-7b-chat-hf llama_openvino``
+In this case, you can load the converted model in OpenVINO representation directly from the disk:
```
Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:94]:
```suggestion
Alternatively, you can download and convert the model using CLI interface: ``optimum-cli export openvino --model meta-llama/Llama-2-7b-chat-hf llama_openvino``.
In this case, you can load the converted model in OpenVINO representation directly from the disk:
```

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+
+
+By default, inference will run on CPU. To select a different inference device, for example GPU, add `device="GPU"` to the `.from_pretrained()` call. To switch to a different device after the model has been loaded, use the `.to()` method. The device naming convention is the same as in OpenVINO native API:
+
+.. code-block:: python
+
+    model.to("GPU")
+
+
+Optimum-Intel API also provides out-of-the-box model optimization through weights compression using NNCF which substantially reduces the model footdpring and inference latency:
```
Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:109]: not sure but I checked that Optimum Intel is spelled as two words in other docs, correct me if that's wrong

Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:109]:
```suggestion
Optimum Intel API also provides out-of-the-box model optimization through weight compression using NNCF which substantially reduces the model footprint and inference latency:
```

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+
+.. code-block:: python
+
+    model = OVModelForCausalLM.from_pretrained(model_id, export=True, load_in_8bit=True)
+
+
+Wieght compression is applied by default to models larger than 1 billion parameters and also available for CLI interface as ``--int8`` option.
```
Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:116]:
```suggestion
Weight compression is applied by default to models larger than one billion parameters and is also available for CLI interface as the ``--int8`` option.
```

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+
+Working with models tuned with LoRA
```
Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:118]:
```suggestion
Working with Models Tuned with LoRA
```

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 10]
```diff
@@ -0,0 +1,165 @@
...
+++++++++++++++++++++++++++++++++++++
+
+Low-ranking Adaptation (LoRA) is a popular method to tune Generative AI models to a downstream task or custom data. But it requires some extra steps to be done for efficient deployment using the Hugging Face API. Namely, the trained adapters should be fused into the baseline model to avoid extra computation. This is how it can be done for LLMs:
```
Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:121]:
```suggestion
Low-rank Adaptation (LoRA) is a popular method to tune Generative AI models to a downstream task or custom data. However, it requires some extra steps to be done for efficient deployment using the Hugging Face API. Namely, the trained adapters should be fused into the baseline model to avoid extra computation. This is how it can be done for Large Language Models (LLMs):
```

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 11]
```diff
@@ -0,0 +1,165 @@
...
+
+    model = AutoModelForCausalLM.from_pretrained(model_id, use_cache=True)
+    model = PeftModelForCausalLM.from_pretrained(model, lora_adaptor)
+    model.merge_and_unload()
+    model.get_base_model().save_pretrained("fused_lora_model")
+
+
+Now the model can be converted to OpenVINO using Optimum-Intel Python API or CLI interfaces mentioned above.
+
+Running Generative AI models using native OpenVINO APIs
```
Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:136]:
```suggestion
Running Generative AI Models using Native OpenVINO APIs 
```

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 12]
```diff
@@ -0,0 +1,165 @@
...
+    model_id = "meta-llama/Llama-2-7b-chat-hf"
+    lora_adaptor = "./lora_adaptor"
+
+    model = AutoModelForCausalLM.from_pretrained(model_id, use_cache=True)
+    model = PeftModelForCausalLM.from_pretrained(model, lora_adaptor)
+    model.merge_and_unload()
+    model.get_base_model().save_pretrained("fused_lora_model")
+
+
+Now the model can be converted to OpenVINO using Optimum-Intel Python API or CLI interfaces mentioned above.
```
Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:134]:
```suggestion
Now the model can be converted to OpenVINO using Optimum Intel Python API or CLI interfaces mentioned above.
```

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 13]
```diff
@@ -0,0 +1,165 @@
...
+
+Running Generative AI models using native OpenVINO APIs 
+########################################################
+
+To run Generative AI models using native OpenVINO APIs you need to follow regular **Сonvert -> Optimize -> Deploy** path with few simplifications.
```
Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:139]:
```suggestion
To run Generative AI models using native OpenVINO APIs, follow the regular **Сonvert -> Optimize -> Deploy** path with few simplifications. 
```

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 14]
```diff
@@ -0,0 +1,165 @@
...
+
+
+Now the model can be converted to OpenVINO using Optimum-Intel Python API or CLI interfaces mentioned above.
+
+Running Generative AI models using native OpenVINO APIs 
+########################################################
+
+To run Generative AI models using native OpenVINO APIs you need to follow regular **Сonvert -> Optimize -> Deploy** path with few simplifications. 
+
+To convert model from Hugging Face you can use Optimum-Intel export feature that allows to export model in OpenVINO format without invoking conversion API and tools directly, as it is shown above. In this case, the conversion process is a bit more simplified. You can still use regular conversion path if model comes from outside of Hugging Face ecosystem, i.e., in source framework format (PyTorch, etc.)
```
Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:141]:
```suggestion
To convert a model from Hugging Face, you can use the Optimum Intel export feature, which enables model export in the OpenVINO format without directly invoking conversion API and tools, as shown above. This simplifies the conversion process. You can still use the regular conversion path if the model comes from outside the Hugging Face ecosystem, such as in a source framework format (PyTorch, etc.). 
```

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 15]
```diff
@@ -0,0 +1,165 @@
...
+Now the model can be converted to OpenVINO using Optimum-Intel Python API or CLI interfaces mentioned above.
+
+Running Generative AI models using native OpenVINO APIs 
+########################################################
+
+To run Generative AI models using native OpenVINO APIs you need to follow regular **Сonvert -> Optimize -> Deploy** path with few simplifications. 
+
+To convert model from Hugging Face you can use Optimum-Intel export feature that allows to export model in OpenVINO format without invoking conversion API and tools directly, as it is shown above. In this case, the conversion process is a bit more simplified. You can still use regular conversion path if model comes from outside of Hugging Face ecosystem, i.e., in source framework format (PyTorch, etc.) 
+
+Model optimization could be performed within Hugging Face or directly using NNCF as described :doc:`here <weight_compression>`.
```
Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:143]:
```suggestion
Model optimization can be performed within Hugging Face or directly using NNCF as described in the :doc:`weight compression guide <weight_compression>`.
```

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 16]
```diff
@@ -0,0 +1,165 @@
...
+
+Inference code that uses native API cannot benefit from Hugging Face pipelines, hence you need to write your own or take it from the examples available. Below are few examples of popular Generative AI scenarios:
```
Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:145]:
```suggestion
Inference code that uses native API cannot benefit from Hugging Face pipelines. You need to write your custom code or take it from the available examples. Below are some examples of popular Generative AI scenarios:
```

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 17]
```diff
@@ -0,0 +1,165 @@
...
+
+* In case of text Large Language Models (LLMs), this includes tokenization, inference and token selection loop, and de-tokenization. In case if token selection involves beam search, this also needs to be written.  
+* For Image generation models you need make a pipeline that includes several model: inference for source (e.g. text) encoder models, inference loop for diffusion process and inference for decoding part. Scheduler code is also required.
```
Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:148]:
```suggestion
* For image generation models, you need to make a pipeline that includes several model inferences: inference for source (e.g., text) encoder models, inference loop for diffusion process and inference for decoding part. Scheduler code is also required. 
```

Comment (tsavina) [reply docs/articles_en/openvino_workflow/gen_ai.md:148]: not sure what is meant by (e.g., text) here

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 18]
```diff
@@ -0,0 +1,165 @@
...
+
+Model optimization could be performed within Hugging Face or directly using NNCF as described :doc:`here <weight_compression>`.
+
+Inference code that uses native API cannot benefit from Hugging Face pipelines, hence you need to write your own or take it from the examples available. Below are few examples of popular Generative AI scenarios:
+
+* In case of text Large Language Models (LLMs), this includes tokenization, inference and token selection loop, and de-tokenization. In case if token selection involves beam search, this also needs to be written.  
+* For Image generation models you need make a pipeline that includes several model: inference for source (e.g. text) encoder models, inference loop for diffusion process and inference for decoding part. Scheduler code is also required. 
+
+To write such pipelines, we recommend following examples that we have provided as a part of OpenVINO:
```
Comment (tsavina) [inline docs/articles_en/openvino_workflow/gen_ai.md:150]:
```suggestion
To write such pipelines, you can follow the examples provided as part of OpenVINO: 
```

File: docs/articles_en/openvino_workflow.md [hunk 19]
```diff
@@ -18,6 +18,7 @@
    Deployment on a Local System  <openvino_deployment_guide>
    Deployment on a Model Server <ovms_what_is_openvino_model_server>
    pytorch_2_0_torch_compile
+   Generative AI Optimization and Deployment <gen_ai_guide>
```
[the remark this answers is not in the document]
Comment (kblaszczak-intel) [reply docs/articles_en/openvino_workflow.md:21]: I am convinced we should use the "Learn OpenVINO" section for this kind of content. For now, we can put it in the first level of the section and when we have a couple more, we can create a whole subsection for such guides.

File: docs/home.rst [hunk 20]
```diff
@@ -96,6 +96,13 @@ OpenVINO 2023.0
 
       Optimize generation of the graph model with PyTorch 2.0 torch.compile() backend
 
+   .. grid-item-card:: Generative AI optimization and deployment
+      :link: gen_ai_guide
+      :link-alt: gen ai
+      :link-type: doc
+
+      Generative AI optimization and deployment
+
```
Comment (kblaszczak-intel) [inline docs/home.rst:105]: We should consider removing one of the existing tiles, as too many of them may defeat the purpose of exposing highly important information this way. Maybe "Run Inference" is not necessary anymore, as it is well exposed in the menu, unlike in the 22.3 version.

File: docs/articles_en/openvino_workflow.md (hunk 19 above)
[the remark this answers is not in the document]
Comment (AlexKoff88) [reply docs/articles_en/openvino_workflow.md:21]: I am ok if it is planned in the first level of the "Learn OpenVINO" section

File: docs/home.rst (hunk 20 above)
Comment (AlexKoff88) [reply docs/home.rst:105]: you can reshuffle them afterward. But Gen AI doc is something we would like to give more visibility

**Comment by AlexKoff88** [comment]
Most of the comment is resolved. The doc build is successful.

**Comment by yury-gorbachev** [comment]
Great work, everyone. 
I still have one comment.
Could we add links to notebooks in optimum-intel section? For native API we have links to samples, but for that HF we don't. @eaidova - could we pick some neat notebooks to showcase? Like SD, Chatbot, something very simple...
It would be also nice to give a link to Optimum-intel documentation

Wonder where boost comes from in our dependencies, we were  running from it like from fire, could we remove it specifically?

BTW, I would recommend to check grammar a bit more, have seen at least 2 errors in the text (Wieght, footdpring)

File: docs/articles_en/openvino_workflow/gen_ai.md (hunk 7 above)
Comment (helena-intel) [reply docs/articles_en/openvino_workflow/gen_ai.md:109]: I would keep "weights compression" since it's the name of the method and it should be easily findable with that name.

File: docs/articles_en/openvino_workflow/gen_ai.md (hunk 15 above)
Comment (helena-intel) [reply docs/articles_en/openvino_workflow/gen_ai.md:143]: Link to HF docs for model optimization: https://huggingface.co/docs/optimum/intel/optimization_ov (weights compression will be added soon)

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 21]
```diff
@@ -0,0 +1,165 @@
...
+     - Python (Hugging Face API)
+     - Python, C++ (OpenVINO API)
+   * - Model Format
+     - Source Framework / OpenVINO
+     - OpenVINO
+   * - Inference code
+     - Hugging Face based
+     - Custom inference pipelines
+   * - Additional dependencies
+     - As much as Hugging Face has
```
[the remark this answers is not in the document]
Comment (helena-intel) [reply docs/articles_en/openvino_workflow/gen_ai.md:47]: It is all Hugging Face dependencies. Technically it depends on the model too, for some models you need extra dependencies. But the idea here is to show that (for now at least) the HF path has many dependencies.

File: docs/articles_en/openvino_workflow/gen_ai.md (hunk 7 above)
Comment (ngaloppo) [reply docs/articles_en/openvino_workflow/gen_ai.md:109]: I see Helena's point, but that's not how it's typically referred to in the industry. As far as I can tell, the concept is "weight compression", and the action/verb is "compress weights". In terms of method/API naming I would expect something like `weight_compression()` or `compress_weights()`. If that is not the case today, and there's still time to change it, then I would recommend fixing the API name.

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 22]
```diff
@@ -0,0 +1,165 @@
...
+.. list-table::
+   :widths: 20 25 55
+   :header-rows: 1
+
+   * -  
+     - Hugging Face through OpenVINO
+     - OpenVINO Native API
+   * - Model support
+     - Wide set of Models
+     - Wide set of Models
```
Comment (ngaloppo) [inline docs/articles_en/openvino_workflow/gen_ai.md:36]:
```suggestion
     - Broad set of Models
     - Broad set of Models
```

File: docs/articles_en/openvino_workflow/gen_ai.md (hunk 3 above)
Comment (yury-gorbachev) [reply docs/articles_en/openvino_workflow/gen_ai.md]: I agree- it should not be mentioned, this is part of OV distribution.

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 23]
```diff
@@ -0,0 +1,165 @@
...
+* Inference on CPU and GPU platforms, including Intel integrated and discrete ARC GPUs. 
+ 
+
+OpenVINO offers two main paths for Generative AI use cases:
+
+* Using OpenVINO as a backend for Hugging Face frameworks (transformers, diffusers) through the `Optimum Intel <https://huggingface.co/docs/optimum/intel/inference>`__ extension.
+* Using OpenVINO native APIs (Python and C++) with custom pipeline code. 
+
+ 
+In both cases, OpenVINO runtime and tools are used, the difference is mostly in the preferred API and the final solution's footprint. Native APIs enable the use of generative models in C++ applications, ensure minimal runtime dependencies, and minimize application footprint. The Native APIs approach requires the implementation of glue code (generation loop, text tokenization, or scheduler functions), which is hidden as internal implementation details within Hugging Face libraries.
```
[the remark this answers is not in the document]
Comment (AlexKoff88) [reply docs/articles_en/openvino_workflow/gen_ai.md:19]: "...for a better developer experience." sounds more suitable to me.

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 24]
```diff
@@ -0,0 +1,165 @@
...
+
+    model = OVModelForCausalLM.from_pretrained(model_id, export=True, load_in_8bit=True)
+
+
+Wieght compression is applied by default to models larger than 1 billion parameters and also available for CLI interface as ``--int8`` option.
+
+Working with models tuned with LoRA
+++++++++++++++++++++++++++++++++++++
+
+Low-ranking Adaptation (LoRA) is a popular method to tune Generative AI models to a downstream task or custom data. But it requires some extra steps to be done for efficient deployment using the Hugging Face API. Namely, the trained adapters should be fused into the baseline model to avoid extra computation. This is how it can be done for LLMs:
```
[the remark this answers is not in the document]
Comment (AlexKoff88) [reply docs/articles_en/openvino_workflow/gen_ai.md:121]: I don't think this is good to put too many links to 3rd party docs. It is just yet another way to leave OpenVINO docs. I think this part is useless for those who don't know what LoRA is about.
But let's see what our documentation team says about that. @kblaszczak-intel, @tsavina?

File: docs/articles_en/openvino_workflow/gen_ai.md [hunk 25]
```diff
@@ -0,0 +1,165 @@
...
+.. code-block:: python
+
+    model.to("GPU")
+
+
+Optimum-Intel API also provides out-of-the-box model optimization through weights compression using NNCF which substantially reduces the model footdpring and inference latency:
+
+.. code-block:: python
+
+    model = OVModelForCausalLM.from_pretrained(model_id, export=True, load_in_8bit=True)
```
[the remark this answers is not in the document]
Comment (AlexKoff88) [reply docs/articles_en/openvino_workflow/gen_ai.md:113]: OVQuantizer is still there and it is exposed in the HF docs. Here we who the most simplest way.
`load_in_8bit` is exactly the same interface that already exists in AutoModelForCausalLM. We just added the implementation for OpenVINO in Optimum-Intel.

**Comment by AlexKoff88** [comment]
> Great work, everyone. I still have one comment. Could we add links to notebooks in optimum-intel section? For native API we have links to samples, but for that HF we don't. @eaidova - could we pick some neat notebooks to showcase? Like SD, Chatbot, something very simple... It would be also nice to give a link to Optimum-intel documentation
> 
> Wonder where boost comes from in our dependencies, we were running from it like from fire, could we remove it specifically?
> 
> BTW, I would recommend to check grammar a bit more, have seen at least 2 errors in the text (Wieght, footdpring)

Resolved.

**Comment by helena-intel** [comment]
> I see Helena's point, but that's not how it's typically referred to in the industry. As far as I can tell, the concept is "weight compression"
Then we should use that indeed! But this should be consistent across docs. The main NNCF doc calls it "Weights Compression" (https://github.com/openvinotoolkit/nncf/blob/develop/docs/compression_algorithms/CompressWeights.md) at the moment.

**Review by kblaszczak-intel — APPROVED** [review]
Whatever adjustments are still needed, they can be implemented in future PRs.

File: docs/articles_en/openvino_workflow/gen_ai.md (hunk 24 above)
[the remark this answers is not in the document]
Comment (kblaszczak-intel) [reply docs/articles_en/openvino_workflow/gen_ai.md]: We can add more links at the end of the article, in "Additional Resources." This way we don't distract readers when they should focus on our content but we give them the reference when they are done.

**Review by yury-gorbachev — APPROVED** [review]

File: docs/articles_en/openvino_workflow/gen_ai.md (hunk 25 above)
[the remark this answers is not in the document]
Comment (ngaloppo) [reply docs/articles_en/openvino_workflow/gen_ai.md:113]: Makes sense then, if we're following HF practice. Still feels clumsy to me ;-)

**Comment by ngaloppo** [comment]
Ship it.
